package.json

```diff
@@ -40,6 +40,18 @@
     ]
   },
   "devDependencies": {
-    "cypress": "^8.2.0"
+    "babel-eslint": "^10.1.0",
+    "cypress": "^8.2.0",
```

Comment:
- [OPTIONAL] The requirements for this project don't say anything about doesn't use another tools but, is really necessary to use cypress? Please, consider removing this. 🐱